app/src/main/java/org/mozilla/FirefoxLite/firefoxdownloader/SampleService.java

```diff
@@ -0,0 +1,249 @@
+package org.mozilla.FirefoxLite.firefoxdownloader;
+
+import android.app.Notification;
+import android.app.Service;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.SharedPreferences;
+import android.graphics.Color;
+import android.os.IBinder;
+import android.preference.PreferenceManager;
+import android.util.JsonReader;
+import android.util.Log;
+import android.widget.Toast;
+import androidx.annotation.Nullable;
+import androidx.core.app.NotificationCompat;
+import androidx.lifecycle.LifecycleOwner;
+import androidx.lifecycle.LifecycleService;
+
+import static com.sample.NotificationHelper.channel1ID;
+
+public class SampleService extends LifecycleService {
```

Comment:
Is this service in your library module?

Reply:
@cnevinc Currently this service is not in the library module but in the app module.As explained in the video chat by you to put service in the library part ,I again refactor the code .

Reply:
@cnevinc Shifted to library module,will add the code of  the service when the code for the library is put up on master branch .